processing/src/main/java/org/apache/druid/query/QueryContexts.java

```diff
@@ -46,6 +46,7 @@
   public static final String VECTORIZE_KEY = "vectorize";
   public static final String VECTOR_SIZE_KEY = "vectorSize";
   public static final String JOIN_FILTER_PUSH_DOWN_KEY = "enableJoinFilterPushDown";
+  public static final String JOIN_FILTER_REWRITE_KEY = "enableJoinFilterRewrite";
```

Comment:
Does this need to be added to the docs?

Reply:
It should, I'll do a follow on PR that adds docs for `enableJoinFilterPushDown` as well